src/adapters/adsparc.js

```diff
@@ -0,0 +1,106 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+/**
+ * Adapter for requesting bids from Adsparc.
+ *
+ * @returns {{callBids: _callBids}}
+ * @constructor
+ */
+var AdsparcAdapter = function AdsparcAdapter() {
+    var bids;
+    function _callBids(params) {
+      bids = params.bids || [];
+      for (var i = 0; i < bids.length; i++) {
+        var bid = bids[i];
+
+        _requestBid(bid);
+
+      }
+      // Only make one request per "nid"
+
+    }
+	
+    var getJSON = function(url) {
+      return new Promise(function(resolve, reject) {
```

Comment:
You cannot use Promises from within Prebid.js.  Currently the build does not use babel-polyfill (which includes Promises) and Promises are not supported within Internet Explorer.

I'm also not sure on Prebid.js' support of Cross-origin AJAX.  Due to supporting IE9, you'd have to use the XDomainRequest object at the very least as regular CORs is not allowed within the XMLHttpRequest object in IE9.

If you would like to use JSONP for x-domain requests, there are helper functions here: https://github.com/prebid/Prebid.js/blob/master/src/adloader.js